tests/micro/project_api/test_project_api.py

```diff
@@ -40,15 +42,36 @@
 
```

Comment:
Is there any test that shows how to pass the west_cmd if one doesn't use the default one? If there is not, it would be nice to add one.

Reply:
`west_cmd` can be passed as a project option, we also had option to pass it as pytest argument before but since project api uses the west from ENV, we removed that